app/paths.js

```diff
@@ -7,10 +7,10 @@ module.exports = {
     index: '/'
   },
   pay: {
-    product: '/pay/:productExternalId'
+    product: '/pay/:productExternalId',
+    complete: '/payment-complete/:paymentExternalId'
```

Comment:
I thought the idea was to have both the `:productExternalId` and the `:paymentExternalId` in the url. So that each payment only needed be unique tot he product, not globally unique.

Reply:
Thats for the user friendly reference. This is the original randomUuid that we have on payments which is unique across all apyments

Reply:
Should we not be using user friendly URLs whenever possible so:
`/:user-freindly-product-reference/payment-complete/:user-freindly-payment-reference`?

Reply:
is that really necessary? (Note we don't have a user friendly product reference, just the payment reference) given that this is part of the URL and most our references in URLs takes this form.

Reply:
Yeah I think it might be a `nice-to-have` rather than a `must-have`. 👍